src/controllers/index.js

```diff
@@ -4,10 +4,12 @@ const signup = require('./signup');
 const specialize = require('./specialization');
 const login = require('./login');
 const error = require('./error');
+const authorizatin = require('../middlewares/authorization');
 
 const router = express.Router();
 
 router.get('/login', login.get);
+router.use(authorizatin);
```

Comment:
typo
```suggestion
router.use(authorization );
```

Reply:
edit both the module name (alias) and name of the file

Reply:
@knanahassouna1  done for the alias name